meta/applications/yc_s15.md

```diff
@@ -0,0 +1,466 @@
+# YC Application
+
+hackEDU's nonprofit YC application for S15.
+
+### Application
+
+**Organization name**
+
+hackEDU
+
+**Organization url, if any**
+
+https://hackedu.us
+
+**If you have an online demo, what's the url?**
+
+N/A
+
+**What will your organization do?**
+
+hackEDU will create a programming club at every high school in America.
+
+We think it's tragic that students who love music can join band, students who
```

Comment:
This reads "we think it's tragic that students who love music can join band".  I think adding a "while" in there might help.